src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -75,6 +75,106 @@ inline int fastrand()
     return (g_seed>>16)&0x7FFF;
 }
 
+inline void rpp_host_rng_xorwow_f32_initialize_8seed_stream(RpptXorwowState *xorwowInitialState, Rpp32u seed)
+{
+    Rpp32u xorwowSeedStream[8];
+    xorwowSeedStream[0] = seed + 1436017U;
+    xorwowSeedStream[1] = seed + 2316719U;
+    xorwowSeedStream[2] = seed + 4377713U;
+    xorwowSeedStream[3] = seed + 1648031U;
+    xorwowSeedStream[4] = seed + 1194655U;
+    xorwowSeedStream[5] = seed + 2224453U;
+    xorwowSeedStream[6] = seed + 3005991U;
+    xorwowSeedStream[7] = seed + 2790345U;
+    for (int i = 0; i < 8; i++)
+    {
+        xorwowInitialState[i].x[0] = 123456789U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[1] = 362436069U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[2] = 521288629U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[3] = 88675123U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[4] = 5783321U + xorwowSeedStream[i];
+        xorwowInitialState[i].counter = 6615241U + xorwowSeedStream[i];
+    }
+}
+
+inline void rpp_host_rng_xorwow_f32_initialize_4seed_stream(RpptXorwowState *xorwowInitialState, Rpp32u seed)
+{
+    Rpp32u xorwowSeedStream[4];
+    xorwowSeedStream[0] = seed + 1436017U;
+    xorwowSeedStream[1] = seed + 2316719U;
+    xorwowSeedStream[2] = seed + 4377713U;
+    xorwowSeedStream[3] = seed + 1648031U;
+    for (int i = 0; i < 4; i++)
+    {
+        xorwowInitialState[i].x[0] = 123456789U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[1] = 362436069U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[2] = 521288629U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[3] = 88675123U + xorwowSeedStream[i];
+        xorwowInitialState[i].x[4] = 5783321U + xorwowSeedStream[i];
+        xorwowInitialState[i].counter = 6615241U + xorwowSeedStream[i];
+    }
+}
+
+inline __m256 rpp_host_rng_xorwow_8_f32_avx(__m256i *pxXorwowStateXParam, __m256i *pxXorwowStateCounterParam)
+{
+    __m256i px362437 = _mm256_set1_epi32(362437);
+    __m256i pxFFFFFFFF = _mm256_set1_epi32(0xFFFFFFFF);
+    __m256i px7FFFFF = _mm256_set1_epi32(0x7FFFFF);
+    __m256i pxExponentFloat = _mm256_set1_epi32(0b111111100000000000000000000000);
```

Comment:
please use hex value (0x) for set1_epi32

Reply:
Done